core/src/avm2/globals/flash/geom/transform.rs

```diff
@@ -173,6 +205,42 @@ pub fn color_transform_to_object<'gc>(
     Ok(object.into())
 }
 
+pub fn matrix3d_to_object<'gc>(
```

Comment:
Just a nitpick: I think `matrix3d_to_object` and `object_to_matrix3d` don't have to be public